src/main/java/com/microsoft/sqlserver/jdbc/ISQLServerCallableStatement.java

```diff
@@ -8,49 +8,1086 @@
+     * @return the column value; if the value is SQL NULL, the value returned is null
+     * @throws SQLServerException
+     *             when an error occurs
+     */
+    public Timestamp getSmallDateTime(int index) throws SQLServerException;
+
+    /**
+     * Retrieves the value of the designated column in the current row of this ResultSet object as a java.sql.Timestamp object in the Java programming
+     * language.
+     * 
+     * @param parameterName
+     *            The name of a column.
+     * @return the column value; if the value is SQL NULL, the value returned is null
+     * @throws SQLServerException
+     *             when an error occurs
+     */
+    public Timestamp getSmallDateTime(String parameterName) throws SQLServerException;
+
+    /**
+     * Retrieves the value of the designated column in the current row of this ResultSet object as a java.sql.Timestamp object in the Java programming
+     * language.
+     * 
+     * @param index
+     *            the first column is 1, the second is 2, ...
+     * @param cal
+     *            the java.util.Calendar object to use in constructing the smalldateTime
+     * @return the column value; if the value is SQL NULL, the value returned is null
+     * @throws SQLServerException
+     *             when an error occurs
+     */
+    public Timestamp getSmallDateTime(int index,
+            Calendar cal) throws SQLServerException;
+
+    /**
+     * Retrieves the value of the designated column in the current row of this ResultSet object as a java.sql.Timestamp object in the Java programming
+     * language.
+     * 
+     * @param name
+     *            The name of a column
+     * @param cal
+     *            the java.util.Calendar object to use in constructing the smalldateTime
+     * @return the column value; if the value is SQL NULL, the value returned is null
+     * @throws SQLServerException
+     *             when an error occurs
+     */
+    public Timestamp getSmallDateTime(String name,
+            Calendar cal) throws SQLServerException;
 
     /**
      * Gets the DateTimeOffset value of parameter with index parameterIndex
      * 
      * @param parameterIndex
      *            the first parameter is 1, the second is 2, and so on
      * @return DateTimeOffset value
-     * @throws SQLException
+     * @throws SQLServerException
      *             if parameterIndex is out of range; if a database access error occurs or this method is called on a closed
      *             <code>CallableStatement</code>
      */
-    public microsoft.sql.DateTimeOffset getDateTimeOffset(int parameterIndex) throws SQLException;
+    public microsoft.sql.DateTimeOffset getDateTimeOffset(int parameterIndex) throws SQLServerException;
 
     /**
      * Gets the DateTimeOffset value of parameter with name parameterName
      * 
      * @param parameterName
      *            the name of the parameter
      * @return DateTimeOffset value
```

Comment:
most other return javadocs have a small description of the value, should we add one to this as well to maintain consistency?